ibis-server/tests/routers/v3/connector/postgres/test_query.py

```diff
@@ -136,6 +136,40 @@ async def test_query(client, manifest_str, connection_info):
     }
 
 
+async def test_query_with_cache(client, manifest_str, connection_info):
```

Comment:
I think it's good to add the test in `tests/routers/v3/connector/postgres/test_fallback_v2.py`. To cover the behavior for fallback.